packages/web-components/package.json

```diff
@@ -23,7 +23,7 @@
     "test:watch": "stencil test --spec --e2e --watchAll",
     "test-a11y": "jest",
     "create-component": "stencil generate",
-    "storybook": "npm-run-all --parallel build:watch start-storybook",
+    "storybook": "npm-run-all --parallel build start-storybook",
```

Comment:
Isn't this required if making updates to the components and expecting to see it within storybook?